example/android/build.gradle

```diff
@@ -8,6 +8,8 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:$androidPluginVersion"
+        getReactNativeDependencies().each { dependency ->
+            classpath(dependency)
+        }
```

Comment:
Was this related to upgrading `react-native-test-app`?

Reply:
Yeah, it should make enabling/disabling new arch easier.